src/components/atoms/StoryCard/index.js

```diff
@@ -0,0 +1,95 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { get } from "lodash";
+import { Link } from "@quintype/components";
+import Contributor from "../../atoms/Contributor";
+import ResponsiveImageWithFallback from "../ResponsiveImageWithFallback";
+import Icon from "../Icon";
+
+import { getStoryData, generateImageSources } from "../../utils/utils";
+// import { isPremium } from "../../../../isomorphic/data/story";
+
+import "./styles.m.css";
```

Comment:
can this file be renamed to `storyCard.m.css` ?

Reply:
Done